ext/handlers/filters/messages.go

```diff
@@ -32,6 +33,19 @@ func Text(msg *gotgbot.Message) bool {
 	return msg.Text != ""
 }
 
+func Regex(text string) Message {
+	return func(msg *gotgbot.Message) bool {
+		   if msg.Text == "" {
```

Comment:
Should the regex filter also work on captions?

Reply:
why not